e2e/testcases/stress_test.go

```diff
@@ -637,28 +634,29 @@ func TestStressResourceGroup(t *testing.T) {
 		})
 	}
 
-	err := testresourcegroup.UpdateResourceGroup(nt.KubeClient, rgNN, func(rg *v1alpha1.ResourceGroup) {
-		rg.Spec.Resources = resources
-	})
-	if err != nil {
-		nt.T.Fatal(err)
-	}
+	nt.T.Logf("Update the ResourceGroup spec to add %d ConfigMaps", numConfigMaps)
+	rg.Spec.Resources = resources
+	nt.Must(nt.KubeClient.Update(rg))
```

Comment:
Does this not result in a conflict, given the resourceVersion may have changed since the Create on line 622? I assume the resource-group-controller may have updated the ResourceGroup since then

Reply:
resource-group-controller is paused until the next step sets `status.observedGeneration = metadata.generation`